### PR TITLE
Massive cleanup and changes

### PR DESCRIPTION
**Pull request by gdude2002**
Decided to do some work on the site. Here's a [quick video in Firefox](https://dl.dropboxusercontent.com/u/1547120/ShareX/2016/February/2016-02-21_12-15-41.mp4) to overview the changes. See below for some screenshots as well.

**Firefox**

**Chrome**

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -1,75 +1,245 @@
...
+        <h1>Minecraft's future is bright.</h1>
+        <p class="lead"><b>Meet Glowstone++.</b> It's open, fast, and
+            compatible.</p>
+        <p>Glowstone++ is a completely open-source Minecraft server that
+            doesn't rely on any of Mojang's code - meaning we're free to
+            develop for
+            your needs, and you're free to change it however you want.</p>
+        <a href="https://bamboo.gserv.me/browse/GSPP-SRV/latestSuccessful/artifact/shared/Server-JAR/glowstone%2B%2B-1.8.9-SNAPSHOT.jar"
+           class="btn btn-primary"><i class="fa fa-download"></i> Download for
+            1.8.9 (Experimental)</a>
```
Comment (mastercoms) [inline index.html]: How does this look if you wrap experimental in `small` tags? (without parentheses)

File: index.html [hunk 2]
```diff
@@ -1,75 +1,245 @@
...
-  </body>
+
+<footer class="bg-inverse">
+    <div class="container">
+        <br/>
+        <p>
+            <small>Made with ♥ by some nerds. Our project is licensed under the
+                MIT license. Glowstone++ would not be possible without <a
+                        href="https://github.com/GlowstonePlusPlus/GlowstonePlusPlus#credits">these
+                    folks</a> - thanks for your time and passion!
```
Comment (mastercoms) [inline index.html]: `&mdash;`, not `-`

File: index.html (hunk 1 above)
Comment (gdude2002) [inline index.html]: Looks kinda like this.

Comment (gdude2002) [inline index.html]: Forgot to remove brackets.

**Comment by mastercoms** [comment]
This looks really nice and makes our website a lot more mature! I guess the next step is to get some forums. I'll what I can do about that, unless you have some ideas of your own.

**Comment by gdude2002** [comment]
Thanks - I was kinda basing it off a previous site I did for a plugin, realised it would be useful to have the table there and yeah, did some misc cleanup.

Up to you about the forum - I can host that if you like, but it's up to you how you want to do that, and what software you want.

Would be easiest for me if the software wasn't PHP, though.

File: index.html (hunk 1 above)
Comment (mastercoms) [inline index.html]: Do you think that looks better? And does it look better with a lowercase `e`?

Comment (gdude2002) [inline index.html]: I prefer it with a capital E, though it'd be better if it was smaller than that if we were going that route.

**Comment by mastercoms** [comment]
I was thinking about using [Flarum](http://flarum.org/), and I would be very grateful if you hosted it.

**Comment by gdude2002** [comment]
> Flarum is currently in beta and should not be used in production.

Well, I could set it up if you really want it I guess. What domain would this be going under?

**Comment by mastercoms** [comment]
I don't think we're doing anything where stability is critical, but if it requires more maintenance for you, I can find something else. I believe we'll talk to @SpaceManiac about getting the glowstone.net domain. If not, I'll buy another domain.

**Comment by gdude2002** [comment]
Shall I put it up at `forums.glowstone.net` for now maybe? Or shall I leave that as a redirect to the gs++ site?

**Comment by mastercoms** [comment]
Putting it up on forums.glowstone.net sounds good.

File: index.html (hunk 1 above)
Comment (mastercoms) [inline index.html]: How about if `Experimental` was in italics?

Comment (mastercoms) [inline index.html]: Meh, I like it just small.

**Pull request merged**